doc/index.html

```diff
@@ -769,51 +767,95 @@ <h3 id="validators-list">Validators list</h3>
             These validators are shipped in <code>parsley.js</code>. Have a look at the <a href="#remote">Parsley Remote</a> plugin and <a href="#extras">Parsley Extras</a> for more validators.
           </p>
 
-          <h3 id="validators-craft">Craft yours!</h3>
+          <h1 id="custom">Custom Validators</h3>
+          <h3 id="custom-intro">Craft yours</h3>
           <p>Of course, Parsley built-in validators are commonly used validators, and you'll need some more that fit your specific forms and validations. That's why Parsley lets you easily create your own validators.</p>
           <p>
-          <p>Here again, like localizations, configuring your custom validators and error messages comes in two flavors:
```

Comment:
No more support for *local* validator, because it never actually worked, there was only a single global dictionary of validators. I think local validators are overkill, but they could be rolled in if you think it's helpfull.